rules/experimental/rbe/README.md

```diff
@@ -0,0 +1,259 @@
+# exec_properties.bzl
+
+exec_properties.bzl contains the following Starlark macros:
+* rbe_exec_properties
+* custom_exec_properties
+* create_exec_properties_dict
+* merge_dicts
+
+## rbe_exec_properties
+
+`rbe_exec_properties` is a Starlark macro that can be called from the WORKSPACE file. It wraps a
```

Comment:
`WORKSPACE` (also in code) here and below

Reply:
done.
And `BUILD` too.